tests/lib/newlib/newlib_hook/src/main.c

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2021 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <sys/unistd.h>
+#include <sys/signal.h>
+#include <sys/stat.h>
+
+/**
+ * @brief Verify _open() stub in newlib libc-hooks.c
+ *
+ * @details The compiler warning when call open(),
+ * use fopen() instead.
```

Comment:
That is just because you did not include `fcntl.h`.

Reply:
Yes, thank you! The head file works.